rules/javascript/react/dangerously_set_inner_html.yml

```diff
@@ -4,16 +4,17 @@ patterns:
     filters:
       - not:
           variable: DATA
-          detection: javascript_react_dangerously_set_inner_html_sanitzed_input
+          detection: javascript_react_dangerously_set_inner_html_sanitized_input
 auxiliary:
-  - id: javascript_react_dangerously_set_inner_html_sanitzed_input
+  - id: javascript_react_dangerously_set_inner_html_sanitized_input
     patterns:
       - pattern: |
           $<STRING_LITERAL>
         filters:
           - variable: STRING_LITERAL
             detection: string_literal
             contains: false
+      - $<_>.sanitize($<_>)
```

Comment:
Can we change this to use the shared rule? I'm not sure if `sanitizeHTML` is a typo (shared rule is using `sanitizeHtml`) or if we need that case too.

Reply:
I'll do that in a separate PR